source/includes/bank_accounts.md

```diff
@@ -0,0 +1,328 @@
+  "bank_name": "CRB",
+  "kind": "external",
+  "created_at": 1629916884,
+  "updated_at": 1629916884
+}
+```
+
+This API adds a bank account to an enrollment via Highline. By passing basic bank account information Highline will return the bank account structure to the user based on the user id.
+
+### HTTP Request
+
+`POST http://api.highline.co/users/{id}/bank_accounts`
+
+### Parameters
+
+Name | Type | In | Description
+--------- | ------- | ------- | ------
+`id` *required* | string | path | User id
+`account_number` *required* |string | body | Account number
+`routing_number` *required* | string | body | Routing number
+`account_type` *required* | string | body | Type of account (checking, savings, etc)
+`bank_name` *required* | string | body | Name of bank
+
+### Response
+
+Parameter | Type | Description
+--------- | ------- | -----------
+`id` | string | Bank account ID
+`user_id` | string | User ID
+`user` | json | User structure
+`account_number` | string | Account number (masked)
+`routing_number` | string | Routing number 
+`account_type` | string | Account Type ("checking" "savings" "other")
+`bank_name` | string | Bank name
+`kind` | string | Kind of account ("internal" "virtual" "external")
+`created_at` | timestamp | Date of creation
+`updated_at` | timestamp | Date of update
+
+
+
+
+
+## Get user's bank accounts
+
+```shell
+curl -X GET http://api.highline.co/users/c8e1f36e-c4.../bank_accounts \
+  -H "Authorization Bearer: AccessToken" \
+  -H "Content-Type: application/json"
+```
+
+> Response
+
+```json
+[{
+  "id": "c8e1f36e-c4...",
+  "user_id": "07662260-b7...",
+  "user": {},
+  "account_number": "***2345",
+  "routing_number": "***4321",
+  "account_type": "Checking",
+  "bank_name": "CRB",
+  "kind": "external",
+  "created_at": 1629916884,
+  "updated_at": 1629916884
+}]
+```
+
+This API returns user's bank accounts based on the specified user id.
+
+### HTTP Request
+
+`GET http://api.highline.co/users/{id}/bank_accounts`
+
+### Parameters
+
+Name | Type | In | Description
+--------- | ------- | ------- | ------
+`id` *required* | string | path | User id
+
+### Response
+
+Parameter | Type | Description
+--------- | ------- | -----------
+`id` | string | Bank account ID
+`user_id` | string | User ID
+`user` | json | User structure
+`account_number` | string | Account number (masked)
+`routing_number` | string | Routing number (masked)
+`account_type` | string | Account Type ("checking" "savings" "other")
+`bank_name` | string | Bank name
+`kind` | string | Kind of account ("internal" "virtual" "external")
+`created_at` | timestamp | Date of creation
+`updated_at` | timestamp | Date of update
+
+
+
+
+
+## Get user's bank account
+
+```shell
+curl -X GET http://api.highline.co/users/c8e1f36e-c4.../bank_accounts/d938f36e-c4... \
+  -H "Authorization Bearer: AccessToken" \
+  -H "Content-Type: application/json"
+```
+
+> Response
+
+```json
+{
+  "id": "d938f36e-c4...",
+  "user_id": "c8e1f36e-c4...",
+  "user": {},
```

Comment:
ideally, we return the user object here in the same response

Reply:
in that particular case, we don't return users (no preload), so this object is going to be empty

Reply:
Then, can we remove it here and add omitempty in the json.